doc/releases/quincy.rst

```diff
@@ -5,6 +5,1471 @@ Quincy
 Quincy is the 17th stable release of Ceph. It is named after Squidward
 Quincy Tentacles from Spongebob Squarepants.
 
+
+
+v17.2.8 Quincy
+==============
+
+This is the eighth, and expected to be last, backport release in the Quincy series. We recommend
+that all users update to this release.
+
+Notable Changes
+---------------
+
+Users should expect to see the el8 rpm subdirectory empty and the "dnf" commands are expected
+to fail with 17.2.8.
+They can choose to use 17.2.8 RPM packages for centos 8/el8 provided by CERN as a community
+member or continue to stay at 17.2.7 following instructions
+from https://docs.ceph.com/en/latest/install/get-packages/#rhel, the ceph.repo file should
+point to https://download.ceph.com/rpm-17.2.7/el8 instead of https://download.ceph.com/rpm-quincy/el8
+
+These CERN packages come with no warranty and have not been tested. The software in them has been
+tested by Ceph according to [platforms](https://docs.ceph.com/en/latest/start/os-recommendations/#platforms).
+The repository for el8 builds is hosted by CERN on [Linux @ CERN](https://linuxsoft.cern.ch/repos/ceph-ext-quincy8el-stable/).
+The public part of the GPG key used to sign the packages is available at
+[RPM-GPG-KEY-Ceph-Community](https://linuxsoft.cern.ch/repos/RPM-GPG-KEY-Ceph-Community).
+
+* RADOS: `get_pool_is_selfmanaged_snaps_mode` C++ API has been deprecated
+  due to being prone to false negative results.  Its safer replacement is
+  `pool_is_in_selfmanaged_snaps_mode`.
+* RBD: When diffing against the beginning of time (`fromsnapname == NULL`) in
+  fast-diff mode (`whole_object == true` with the `fast-diff` image feature enabled
+  and valid), diff-iterate is now guaranteed to execute locally if exclusive
+  lock is available.  This brings a dramatic performance improvement for QEMU
+  live disk synchronization and backup use cases.
+* RBD: The option ``--image-id`` has been added to the `rbd children` CLI command,
+  so it can be run for images in the trash.
+* RBD: `RBD_IMAGE_OPTION_CLONE_FORMAT` option has been exposed in Python
+  bindings via the `clone_format` optional parameter to `clone`, `deep_copy` and
+  `migration_prepare` methods.
+* RBD: `RBD_IMAGE_OPTION_FLATTEN` option has been exposed in Python bindings via
+  `flatten` optional parameter to the `deep_copy` and `migration_prepare` methods.
+
+Changelog
+---------
+
+- <span></span>.github: sync the list of paths for rbd label, expand tests label to qa/\* ([pr#57726](https://github.com/ceph/ceph/pull/57726), Ilya Dryomov)
```

Comment:
The changelog is full of `<span>`, `</span>` and `\` (escape) noise in various combinations, which is there on the rendered page as well.  Also, we have never had blank lines between changelog entries before.

Was there a change made to the changlog generation script?

Reply:
I don't know, but added a tracker for it https://tracker.ceph.com/issues/68987
@neha-ojha @ljflores ?

Reply:
@idryomov it was a mistake and I belive it's fixed now, pls take a look